src/Resources/LighthouseReport.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace OhDear\PhpSdk\Resources;
+
+class LighthouseReport extends ApiResource
+{
+    public int $id;
+
+    public int $performanceScore;
```

Comment:
The score can also be `null` (this happens when some Lighthouse audits fail completely)

Reply:
Is that the same for all scores? 👀

Reply:
Yup!

Reply:
Updated, used `?` nullable syntax as we have to support PHP 7.4. Is there a plan to update to PHP 8.1 and higher or something at some point?

Also, made the time ms values nullable, but not sure if they should be as well?